setup.php

```diff
@@ -110,7 +110,8 @@ function plugin_init_example() {
    $PLUGIN_HOOKS['item_empty']['example']      = array('Computer' => 'plugin_item_empty_example');
 
    // Restrict right
-   $PLUGIN_HOOKS['item_can']['example']        = ['Computer' => ['PluginExampleComputer', 'restrict']];
+   $PLUGIN_HOOKS['item_can']['example']          = ['Computer' => ['PluginExampleComputer', 'item_can']];
```

Comment:
As far as I can see, there is no `PluginExampleComputer::item_can()` method; `PluginExampleComputer::restrict()` should be used or renamed.